packages/create-ponder/vite.config.ts

```diff
@@ -10,5 +10,6 @@ export default defineConfig({
   },
   test: {
     globalSetup: ["src/_test/globalSetup.ts"],
+    testTimeout: 60_000,
```

Comment:
this seems very high for tests. why do we need a whole minute here?

Reply:
i usually prefer having the tests fail fast if theres something hanging or something that would cause it to take this long

Reply:
Good point. I increased this mostly because they were timing out consistently on my local Windows VM, and I see them timeout occasionally in CI.

These tests are slow for two reasons.
1. They make HTTP requests to Etherscan to fetch ABIs and start blocks. If the user has not provided an etherscan API key, we wait 5 seconds between each request to avoid getting rate limited. (But in CI, we do provide the API key so it's pretty fast).
2. They run `pnpm install` in the generated ponder project, which can be slow, esp if building `better-sqlite3` from source.

These tests rarely fail for any reason other than a timeout (we don't often make changes to `create-ponder`. So on balance I think it's better to set a high timeout simply to save us the annoyance of having the retry them to get a green CI run.

Reply:
makes sense! in the future we may want to consider having some tests have a smaller time out to increase local development feedback loop time, but thats pretty minor